.github/workflows/report.yml

```
@@ -0,0 +1,57 @@
+name: "Test generation of comparison reports"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  test:
+    name: "Run Tests"
+    uses: ./.github/workflows/test-all.yml
+
+
+  report:
+    name: "Create Report"
+    runs-on: ubuntu-latest
+    needs: test
+    if: ${{ always() }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Python 3.7"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: "Create data and gh-pages worktree"
+        run: |
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}
+          git fetch
+          git branch --track data origin/data
+          git worktree add data origin/data
```

Do you perhaps mean
```suggestion
          git worktree add data data
```
Otherwise there is no point in the --track in the line before...